regulations/static/regulations/css/less/mixins.less

```diff
@@ -33,27 +33,27 @@ mixins.less contains any mixins or variables to be used throughout the LESS file
 @red_orange_80: #D96A4F;
 @red_orange_50: #E9A193;
 
-/* 
+/*
  Type Variables/Mixins
  ----------------------
 */
 
 @body-font: Georgia, serif;
- 
+
 .avenir-next {
-    font-family: "Avenir Next", Arial, sans-serif;
+    font-family: "AvenirNextLTW01-Regular", Arial, sans-serif;
     font-weight: normal;
     font-style: normal;
 }
 
 .avenir-next-demi {
-    font-family: "Avenir Next Demi", Arial, sans-serif;
+    font-family: "AvenirNextLTW01-Demi", Arial, sans-serif;
     font-weight: normal;
     font-style: normal;
 }
 
 .avenir-next-demi-italic {
-    font-family: "Avenir Next Demi Italic", Arial, sans-serif;
+    font-family: "AvenirNextLTW01-DemiItalic", Arial, sans-serif;
     font-weight: normal;
```

Comment:
Good catch! I went what was off our font file before, so it's a good thing we're updating.